tests/collections/test_collections.py

```diff
@@ -38,6 +38,9 @@
     Variable = None
 
 
+pytestmark = pytest.mark.usefixtures("tape_mode")
```

Comment:
Forces all functions in this test file to be run in both tape and tape mode

Reply:
Can you explain what setting the variable `pytestmark` does? It looks so strange...

Reply:
I'm not too sure, I imagine pytest does some weird introspection, and looks for this module variable? If it exists, it applies the mark to all objects in the file. I just learnt to do it from the pytest docs 😆